bin/analysis.cpp

```diff
@@ -115,6 +115,8 @@ class chainNames
 			TTchainNames.push_back(dataTag + "_RunC");
 			TTchainNames.push_back(dataTag + "_RunD_v3");
 			TTchainNames.push_back(dataTag + "_RunD_v4");
+		} if(mode.find("WRto")!=std::string::npos){
```

Comment:
The vector of modes was expanded before the getChainNames call and thus the modes now correspond to the signal names.